docs/cluster-registration.md

```diff
@@ -0,0 +1,63 @@
+<!-- START doctoc generated TOC please keep comment here to allow auto update -->
+<!-- DON'T EDIT THIS SECTION, INSTEAD RE-RUN doctoc TO UPDATE -->
+**Table of Contents**  *generated with [DocToc](https://github.com/thlorenz/doctoc)*
+
+- [Joining Clusters](#joining-clusters)
+- [Checking status of joined clusters](#checking-status-of-joined-clusters)
+- [Unjoining clusters](#unjoining-clusters)
+- [Joining additional clusters in a namespace scoped deployment](#joining-additional-clusters-in-a-namespace-scoped-deployment)
+
+<!-- END doctoc generated TOC please keep comment here to allow auto update -->
+
+# Joining Clusters
+
+You can use the `kubefedctl` tool to join clusters as follows.
+
+```bash
+kubefedctl join cluster1 --cluster-context cluster1 \
+    --host-cluster-context cluster1 --v=2
+kubefedctl join cluster2 --cluster-context cluster2 \
+    --host-cluster-context cluster1 --v=2
+```
+
+Repeat this step to join any additional clusters.
+
+**NOTE:** `cluster-context` will default to use the joining cluster name if not
+specified.
+
+# Checking status of joined clusters
+
+Check the status of the joined clusters by using the following command.
+
+```bash
+kubectl -n kube-federation-system get kubefedclusters
+
+NAME       READY   AGE
+cluster1   True    1m
+cluster2   True    1m
+
+```
+# Unjoining clusters
+
+You can unjoin clusters using `kubefedctl` tool as follows.
+
+```bash
+kubefedctl unjoin cluster2 --cluster-context cluster2 --host-cluster-context cluster1 --v=2
+```
+Repeat this step to unjoin any additional clusters.
+
+# Joining additional clusters in a namespace scoped deployment
+
+Joining additional clusters to a namespaced control plane requires
+providing additional arguments to `kubefedctl join`:
+
+- `--kubefed-namespace=<namespace>` to ensure the cluster has been registered
+  with the KubeFed control plane running in the specified namespace
+
+To join `mycluster` when `KUBEFED_NAMESPACE=test-namespace` was used for deployment:
```

Comment:
These instructions appear to be for a control plane deployed by script rather than helm.  Consider replacing 'when \`KUBEFED_NAMESPACE=test-namespace\` was used for deployment' with 'to a control plane deployed in namespace `test-namespace`'.